.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
```

Comment:
I would not expect husky to be used for pre-commit, just pre-push, pre-commit we want to keep our current solution